README.md

```diff
@@ -1 +1,33 @@
-# Reciate
\ No newline at end of file
+# Reciate
+
+## Our workflow
+
+It is important to us that the specification of this application is forged by consensus among its contributors, and that its implementation follows clearly from the specification. For that reason, everyone who contributes will try, as far as possible, to be rigorous in following this workflow.
+
+If you wish to make a contribution to this repository, we request that you follow these steps:
+
+  1. If the contribution includes a new feature, or changes how a feature will be consumed publicly, please design a specification for the feature or change:
+
+  The process of designing a specification includes
+
+  - Experimenting in code
+
+    Feel free to make changes to the code in a new branch. It is of course difficult to design without the feedback offered by playing with code. Just understand, it is impossible that any changes to the code can be merge into master unless it is in full conformity with the design in the specification, or you can offer a valid justification explaining why it is not (for example, your implementation cuts a corner that would otherwise be very technically challenging, but is still usable; of course, these kinds of divergences from the specification would have to be approved by reviewers).
+
+  - Conversations with other contributors
+
+  - Creating a PR to the README of the master branch of this repository with a description of the specification
+
+  - Thoughtfully considering all of the comments made to the PR
+
+  - Applying the suggested changes to the PR until at least two fellow contributors, including an author of the repository, have accepted the PR, and none are rejecting it.
```

Comment:
Basically, it shouldn't be possible for users to just make PRs and merge them. They need to become 'consensual', and we need a definition of consent. It is impractical to expect every contributor to review every PR before it is merged. On the other hand, it shouldn't be possible for a design change to be approved by 2 or 3 new contributors who may not understand the wider implications. 

Any suggestions?

Reply:
Perhaps we could have a steering committee of core contributors and seek consensus among us. We could review PR and merge them when we have it.

Reply:
How about the following workflow:

Everyone can submit a PR. We will help people get PRs approved. If all core contributors approved a PR it can be merged immediately. 

Once a month (for example every first Sunday) we would have a walk meeting of core contributors to review the state of the project and update priorities, preferably in Journal. 

Perhaps we can also have a short meeting every Sunday, kind of like a daily at KOVnet - no discussions, just quick status update.

What do you think?